jaxley/modules/branch.py

```diff
@@ -95,6 +95,7 @@ def _init_comp_graph(self):
             {
                 "source": list(range(self.ncomp - 1)) + list(range(1, self.ncomp)),
                 "sink": list(range(1, self.ncomp)) + list(range(self.ncomp - 1)),
+                "ordered": list([1] * (self.ncomp - 1)) + list([0] * (self.ncomp - 1)),
```

Comment:
What does ordered do exactly?

Reply:
It indicates whether this compartment edge is along the direction of the xyzr coords.